include/zephyr/sys/uuid.h

```diff
@@ -77,7 +77,7 @@ int uuid_generate_v4(struct uuid *out);
  * @retval -ENOMEM Memory allocation failed
  * @retval -ENOTSUP mbedTLS returned an unrecognized error
  */
-int uuid_generate_v5(const struct uuid *namespace, const void *data, size_t data_size,
+int uuid_generate_v5(const struct uuid *namespace_, const void *data, size_t data_size,
```

Comment:
You also need to update the Doxygen documentation for the parameter accordingly.
Also, I would suggest `ns` instead.

Thanks for the pull request!

Reply:
I thought of using ns too but had no idea what would be preferred.. done